webcomponents/tool/src/main/frontend/js/conversations/sakai-add-topic.js

```diff
@@ -638,6 +644,9 @@ export class SakaiAddTopic extends SakaiElement {
           <input type="button" @click=${this.saveAsDraft} value="${this.i18n.save_as_draft}">
           <input type="button" @click=${this.cancel} value="${this.i18n.cancel}">
         </div>
+        ${this.errorDisplay ? html`
```

Comment:
Is && applicable here?

${this.errorDisplay && html`
           <div id="conv-add-topic-error" class="sak-banner-error">${this.errorDisplay}</div>
`}

Reply:
I'm not actually sure, that might work. I'll try it.